doc/source/cice_index.rst

```diff
@@ -202,6 +202,7 @@ either Celsius or Kelvin units).
    "eps13", "a small number", "10\ :math:`^{-13}`"
    "eps16", "a small number", "10\ :math:`^{-16}`"
    "esno(n)", "energy of melting of snow per unit area (in category n)", "J/m\ :math:`^2`"
+   "eta", "viscous coefficient (shear)", "kg/s"
```

Comment:
Do we also want to point out here that this is actually `2*eta` ? (and same for `zeta` below)